src/dlangui/graphics/glsupport.d

```diff
@@ -694,6 +694,8 @@ bool initGLSupport(bool legacy = false) {
     }
     if (!_glSupport) {
         Log.d("glSupport not initialized: trying to create");
+        int major = to!int(glGetString(GL_VERSION)[0 .. 1]);
```

Comment:
Well, [this page](https://www.khronos.org/opengl/wiki/Get_Context_Info) says, that it is available only on 3.0 and higher.